src/views/Login/index.js

```diff
@@ -134,92 +83,86 @@ export default class Login extends Component {
         // Saves the network state as online in local storage
         localStorage.setItem('network-status', JSON.stringify('online'));
       }
-      this.props.onLogIn();
+      onLogIn();
     } catch (err) {
-      this.setState({ error: err.message, loading: false });
+      setError(err.message);
+      setLoading(false);
     }
-  }
-
-  render() {
-    return (
-      <div className="login">
-        <DocumentTitle title={`Login | ${projectName}`} />
-        <Grid container direction="column" className="container">
-          <img className="login-img" src={GordonLogoVerticalWhite} alt={`${projectName}`} />
-          <form onSubmit={this.logIn}>
-            <TextField
-              id="username"
-              label="Username"
-              placeholder="firstname.lastname"
-              autoComplete="username"
-              value={this.state.username}
-              onChange={this.handleChange('username')}
-              margin="normal"
-              fullWidth
-              autoFocus
-            />
-            <TextField
-              id="password"
-              label="Password"
-              type="password"
-              autoComplete="current-password"
-              value={this.state.password}
-              onChange={this.handleChange('password')}
-              margin="normal"
-              fullWidth
-            />
-            <Typography className="error" variant="body2" color="error">
-              {this.state.error}
-            </Typography>
-            <Grid container justify="center">
-              <Button
-                variant="contained"
-                className="submit-button"
-                type="submit"
-                color="primary"
-                disabled={!this.state.username || !this.state.password || this.state.loading}
-              >
-                {!this.state.loading && 'Log in'}
-                {this.state.loading && <CircularProgress size={24} />}
-              </Button>
-            </Grid>
-          </form>
-        </Grid>
-
-        {this.state.network === 'online' && this.state.showPWALink && (
-          <Grid
-            container
-            justify="center"
-            style={{ margin: '0.5rem' }}
-            onClick={() => {
-              this.setState({ openPWAInstructions: true });
-            }}
-          >
-            <Grid item xs={12} sm={6} md={6} lg={6} xl={6}>
-              <Fab variant="extended" color="primary">
-                <GetAppIcon />
-                <Typography variant="subtitle1">Install Gordon 360</Typography>
-              </Fab>
-            </Grid>
+  };
+
+  return (
+    <div className="login">
+      <DocumentTitle title={`Login | ${projectName}`} />
+      <Grid container direction="column" className="container">
+        <img className="login-img" src={GordonLogoVerticalWhite} alt={`${projectName}`} />
+        <form onSubmit={(e) => logIn(e)}>
```

Comment:
Since you're only passing on the argument, you can just pass `logIn` directly, instead of creating an arrow function callback that passes on the argument to `logIn`:
```suggestion
        <form onSubmit={logIn}>
```